tools/src_kt_runtests.sh

```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
-#
-# Simple script to run Android Kotlin and shell tests locally.
-
-echo "Building java, javatests, and shells/android"
-bazel build \
-  --noshow_progress \
-  --noshow_loading_progress \
-  --test_output=errors \
-  --incompatible_depset_is_not_iterable=false \
-  //java/... //javatests/... //shells/android/...
-TEST_RESULT=$?
-if [$TEST_RESULT != 1];
-then
-  cd -
-  exit ${TEST_RESULT};
-fi
-
-echo "Testing java, javatests, and shells/android"
-bazel test \
-  --noshow_progress \
-  --noshow_loading_progress \
-  --test_output=errors \
-  --incompatible_depset_is_not_iterable=false \
-  //java/... //javatests/... //shells/android/javatests/...
-TEST_RESULT=$?
-
-exit $TEST_RESULT
```

Comment:
Did you mean to delete this file? It's deletion feels unrelated to the rest of the PR.

Reply:
I deleted it because I felt updating it to use bazelisk was strange, given it was already broken :-/

Reply:
Ahhh, gotcha.